### PR TITLE
Backend profile fixes inc. badge updates

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: src/shared/SimpleAccordion/SimpleAccordion.js [hunk 1]
```diff
@@ -5,9 +5,16 @@ import Icon from '../Icon';
 import styles from './SimpleAccordion.css'; // eslint-disable-line no-unused-vars
 
 
-export default ({ title, children }) => (
-  <details styleName="styles.details">
-    <summary><span>{title}</span> <Icon value="chevron-accordion" size={10} /></summary>
-     <div>{children}</div>
-  </details>
-)
+const SimpleAccordian = ({title, children, show = true}) => {
```
Comment (djrobstep) [inline src/shared/SimpleAccordion/SimpleAccordion.js]: spelling?

Comment (maxious) [inline src/shared/SimpleAccordion/SimpleAccordion.js]: fixed

File: src/bundles/CaseStudy/components/DomainList/DomainList.js [hunk 2]
```diff
@@ -236,10 +236,12 @@ class DomainList extends BaseForm {
           if (subMatch.params.id && subMatch.params.id !== 'undefined') {
             // If `id` is present, load from pre-saved state.
             currentStudy = caseStudyForm.case_studies[subMatch.params.id];
+          } else {
+            debugger;
```
Comment (matt-sm) [inline src/bundles/CaseStudy/components/DomainList/DomainList.js]: probably dont want this

Comment (maxious) [inline src/bundles/CaseStudy/components/DomainList/DomainList.js]: true!

**Pull request merged**